src/cljs/ataru/hakija/try_selection.cljs

```diff
@@ -10,13 +10,13 @@
                      (not (clojure.string/blank? answer-id)) (str "&answer-id=" answer-id)
                      selection-id (str "&selection-id=" selection-id))
         c    (async/chan 1)
-        send (fn [selection]
-               (async/put! c [(boolean (:selection-id selection)) [] [selection]] (fn [_] (async/close! c))))]
+        send (fn [selection errors]
+               (async/put! c [(boolean (:selection-id selection)) errors [selection]] (fn [_] (async/close! c))))]
```

Comment:
Tässä pitäis tarkistaa sisältääkö `selection` avaimen `:limit-reached` ja jos ei, niin putata `[false errors []]`.